src/token/types/Token18.sol

```diff
@@ -106,6 +106,25 @@ library Token18Lib {
         IERC20(Token18.unwrap(self)).safeTransferFrom(benefactor, recipient, UFixed18.unwrap(amount));
     }
 
+    /**
+     * @notice Processes a token transfer based on the sign of the amount
+     * @dev If amount is positive, pulls tokens from the account to the caller
+     *      If amount is negative, pushes tokens from the caller to the account
+     * @param self Token to transfer
+     * @param account Address to pull from or push to
+     * @param amount Signed amount of tokens to transfer
+     */
+    function update(Token18 self, address account, Fixed18 amount) internal {
+        int256 sign = amount.sign();
```

Comment:
I think it's less gas if we just compare `amount` to `Fixed18.ZERO` directly, since we end up using `< / >` with `sign` anyways.

Reply:
Agree; `< / >` is noticeably more expensive than `==`.